Frontend/src/pages/registration_edit/components/Refunds.jsx

```diff
@@ -60,7 +65,9 @@ export default function Refunds({ onExit }) {
           ))}
         </Table.Body>
       </Table>
-      <Button onClick={onExit}>Go Back</Button>
+      <Button disabled={isLoading} onClick={onExit}>
```

Comment:
I think the user can still close the modal any time by clicking outside it (I might be misremembering though).

Reply:
it doesn't when using the open={prop}